AnalysisJenkinsfile

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/env groovy
+
+pipeline {
+    agent none
+    environment {
+        COMPLIANCEENABLED = true
+    }
+    options {
+        skipDefaultCheckout()
+        buildDiscarder(logRotator(artifactDaysToKeepStr: '1', artifactNumToKeepStr: '1', daysToKeepStr: '5', numToKeepStr: '10'))
+    }
+    parameters {
+        booleanParam(name: 'WHITESOURCE_SCAN', defaultValue: true, description: 'Run Whitesource scan')
+    }
+    stages {
+        stage ('Run code analysis') {
+            parallel {
+                stage ('Whitesource') {
+                    when {
+                        beforeAgent true
+                        expression { params.WHITESOURCE_SCAN }
+                    }
+                    agent {
+                        docker {
+                            image 'openjdk:8'
```

Comment:
Have you looked at examples at https://propel.ci.build.ge.com/docs/oss-scan/#propel-cli?  Image they are using in the examples: registry.gear.ge.com/dig-propel/propel-oss:latest.
Is it better to use so you don't have to download oss binary?

Reply:
Our `build.gradle` requires setting environment variables for the Artifactory creds. That's not possible if we run it using the OSS image. The job internally creates a new VM using the image specs, so the parent env is not carried over.